bigbluebutton-html5/imports/ui/components/chat/chat-graphql/chat-typing-indicator/component.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { Meteor } from 'meteor/meteor';
+import { useSubscription } from '@apollo/client';
+import {
+  IS_TYPING_PUBLIC_SUBSCRIPTION,
+  IS_TYPING_PRIVATE_SUBSCRIPTION,
+} from '../queries';
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
+import { User } from '/imports/ui/Types/user';
+import Styled from './styles';
+
+interface TypingIndicatorProps {
+  typingUsers: Array<User>,
+  indicatorEnabled: boolean,
+  error: string,
+}
+
+interface TypingIndicatorContainerProps {
+  userId: string,
+  isTypingTo: string,
+  isPrivate: boolean,
+  error: string,
+}
+
+const messages = defineMessages({
+  severalPeople: {
+    id: 'app.chat.multi.typing',
+    description: 'displayed when 4 or more users are typing',
+  },
+});
+
+const CHAT_CONFIG = Meteor.settings.public.chat;
+const TYPING_INDICATOR_ENABLED = CHAT_CONFIG.typingIndicator.enabled;
+
+const TypingIndicator: React.FC<TypingIndicatorProps> = ({
+  typingUsers,
+  indicatorEnabled,
+  error,
+}) => {
+  const intl = useIntl();
+
+  if (!indicatorEnabled || !typingUsers) return null;
```

Comment:
Consider not mount the component in case of not being enable, making it not render the component doesn't mean it's not fetching the data, in that case fetching a data thats not used.